include/kyosu/types/impl/bessel/cb_hn.hpp

```diff
@@ -35,47 +22,39 @@ namespace kyosu::_
   /////////////////////////////////
   // implementation is done using j and y functions
 
-  template<typename Z> KYOSU_FORCEINLINE
-  auto cb_h1_0(Z z) noexcept
-  {
-    return  cb_j0(z)+muli(cb_y0(z));
-  }
+//   template<typename Z> KYOSU_FORCEINLINE
+//   auto cb_h1_0(Z z) noexcept
+//   {
+//     return  cb_j0(z)+muli(cb_y0(z));
+//   }
 
-  template<typename Z> KYOSU_FORCEINLINE
-  auto cb_h2_0(Z z) noexcept
-  {
-    return cb_j0(z)-muli(cb_y0(z));
-  }
+//   template<typename Z> KYOSU_FORCEINLINE
```

Comment:
please remove all comments. Also, this impl/ shoudl probably be in details/ ?

Reply:
done